backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-pvs-csi-doc.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-backing-up-pvs-csi-doc"]
+= Backing up persistent volumes with CSI snapshots
+include::_attributes/common-attributes.adoc[]
+:context: backing-up-applications
+
+toc::[]
+
+You back up persistent volumes with Container Storage Interface (CSI) snapshots by editing the `VolumeSnapshotClass` custom resource (CR) of the cloud storage before you create the `Backup` CR.
+
+.Prerequisites
+
+* The cloud provider must support CSI snapshots.
+* You must enable CSI in the `DataProtectionApplication` CR.
+
+.Procedure
+
+* Add the `metadata.labels.velero.io/csi-volumesnapshot-class: "true"` key-value pair to the `VolumeSnapshotClass` CR:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: <volume_snapshot_class_name>
+  labels:
+    velero.io/csi-volumesnapshot-class: "true"
+driver: <csi_driver>
```

Comment:
maybe an explanation on how to obtain the csi_driver
also, other documents as number tags on the resource followed by a short explanation